.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -76,6 +81,19 @@ on:
           When run on other triggers, attestations are signed and have an "intoto.sigstore" extension.
         value: ${{ jobs.slsa-run.outputs.attestations-download-name }}
 
+      provenance-download-sha256:
+        description: >
+          The sha256 digest of the attestations.
+
+          Users should verify the download against this digest to prevent tampering.
+        value: ${{ jobs.slsa-run.outputs.attestations-download-sha256 }}
+
+      binaries-download-name:
```

Comment:
shall we use `artifacts` instead of `binaries`? What's common in bazel termonology?

Reply:
artifacts is common. switched it.